lib/core/certificates/step.sh

```diff
@@ -53,13 +53,12 @@ function ih::setup::core.certificates::install() {
 
   ih::log::info "Acquiring cert bundle from Mozilla"
   curl https://curl.se/ca/cacert.pem >"$MOZILLA_PATH"
+  # Append our DLP certs to the mozilla bundle.
+  cat "$CA_PATH" >>"$MOZILLA_PATH"
 
-  curl https://curl.se/ca/cacert.pem >"$HOME/.ih/certs/mozilla.pem"
```

Comment:
So as far as I can tell you're just deleting duplicate lines, not actually sorting out the "use $VARIABLE vs. use /full/path" differences?